src/OpenTelemetry.Contrib.Instrumentation.AWS/Implementation/AWSServiceType.cs

```diff
@@ -0,0 +1,35 @@
+// <copyright file="AWSServiceType.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+
+namespace OpenTelemetry.Contrib.Instrumentation.AWS.Implementation;
+
+internal class AWSServiceType
```

Comment:
Is splitting this in a separate file really worth it?

Reply:
With `AWSServiceType` the code looks more self-explanatory. I think tests parameterised with `[InlineData(AWSServiceType.SQSService)]` are looking more elegant. If for If you open the sub-folder you immediately see AWSServiceType and realise what services are covered at the moment. Strongly speaking it's may be had to be an enum, but it brings unnecessary conversions enum <-> string and I decided to keep string constants.